docs/installation/manual/3_configuration.md

```diff
@@ -79,22 +79,17 @@ You do not need to configure the `storage_path` when using a standalone NATS Ser
 In the case that you are connecting to a multi-node NATS cluster, you can configure more than
 one address in the `addresses` field.
 
-## Database connections
+## Database connection using a global connection pool
 
-Configuring database connections varies based on the [database configuration](database)
-that you chose.
-
-### Global connection pool
-
-If you want to use a single connection pool to a single PostgreSQL database, then you must
-uncomment and configure the `database` section within the `global` section:
+If you want to use a single connection pool to a single PostgreSQL database, 
+then you must uncomment and configure the `database` section within the `global` section:
 
 ```yaml
 global:
   # ...
   database:
     connection_string: postgres://user:pass@hostname/database?sslmode=disable
-    max_open_conns: 100
+    max_open_conns: 90
```

Comment:
out of curiosity, why the change in number here? :)

Reply:
As mentioned, the default configuration for Postgres is set to 100, but has reserved 3 slots for administrative usage. So having 90 conns for Dendrite sounds like a fair trade. (Even though we probably could go further down)